embc-app/ClientApp/src/app/volunteer/pages/referral-maker/referral-maker.component.html

```diff
@@ -10,60 +10,88 @@ <h1>Identify Supports</h1>
 
     <section class="clearfix">
       <!-- confirm task start time -->
-      <div class="feature-block card d-block border-0">
-        <div *ngIf="registration?.incidentTask['startDate']">
-          <label>Please confirm timing for commencement of supports:</label>
-          <!-- <div>{{registration.incidentTask['startDate'] | date:'yyyy/MM/dd \'at\' HH:mm:ss a'}}</div> -->
-          <div>Incident start date &amp; time: {{registration.incidentTask['startDate'] | dateTimeFormatPipe}}</div>
+      <div class="card">
+        <div class="card-header">
+          Please confirm timing for commencement of supports:
         </div>
-      </div>
-
-      <!-- select required supports -->
-      <div class="card d-block border-0 p-4">
-        <label>Select required supports:</label>
-        <div class="form-check mt-2">
-          <input type="checkbox" class="form-check-input" [checked]="foodReferrals.length > 0" id="supportFood" (change)="foodReferralChange()">
-          <label class="form-check-label" for="supportFood">Food</label>
-        </div>
-        <div class="form-check mt-2">
-          <input type="checkbox" class="form-check-input" [checked]="clothingReferrals.length > 0" id="supportClothing">
-          <label class="form-check-label" for="supportClothing">Clothing</label>
-        </div>
-        <div class="form-check mt-2">
-          <input type="checkbox" class="form-check-input" [checked]="lodgingReferrals.length > 0" id="supportLodging">
-          <label class="form-check-label" for="supportLodging">Lodging</label>
+        <div class="card-body">
+          <div *ngIf="registration?.incidentTask['startDate']">
+            <!-- <div>{{registration.incidentTask['startDate'] | date:'yyyy/MM/dd \'at\' HH:mm:ss a'}}</div> -->
+            <div>Incident start date &amp; time: {{registration.incidentTask['startDate'] | dateTimeFormatPipe}}</div>
+
+            <div class="row">
+              <div class="col">
+                <span class="align-bottom">
+                  <span class="h3">Support start date &amp; time: {{defaultDate | dateTimeFormatPipe}}</span>
+                </span>
+              </div>
+              <div class="col-2">
+                <span class="align-bottom">
+                  <a href="javascript:void(0)">
+                    <app-font-awesome-icon icon="times"></app-font-awesome-icon>
+                    <span class="ml-2" (click)="showDefaultDatePicker=!showDefaultDatePicker">Override start time</span>
+                  </a>
+                </span>
+              </div>
+              <div class="col-2">
+                <button class="btn btn-primary" (click)="showReferralMakers=!showReferralMakers">NEXT</button>
```

Comment:
Would be ideal (easier flow / less confusing for user) if Next button moved below the override inputs (if they are displayed).
Also, the Next button should disappear once it's clicked.
Also, IMHO the Cancel&Close button and Create Referral button should not be displayed until Next is clicked. (I know this is how the mockups work but IMHO it's horrible and I think N41 thinks so too because they moved the buttons to the "bottom of the page" using spacing... yuck!). Open for discussion of course. AMB can make a decision if needed.

Reply:
Yeah. Charles isn't on board with the interactions of this element. I expect changes from them. I have done a few of your suggestions